### PR TITLE
Update earthquake example

### PR DESCRIPTION
**Pull request by pdeziel**
This updates the earthquake example to work with PyEnsign v0.8b0

### CODE REVIEW

**Review by Aatmaj1 — CHANGES_REQUESTED** [review]
Hi @pdeziel tried running the code in local machine worked successfully and code looks good , I was just wondering should we comment regarding line 10 why it is done? in a single line

**Comment by pdeziel** [comment]
> Hi @pdeziel tried running the code in local machine worked successfully and code looks good , I was just wondering should we comment regarding line 10 why it is done? in a single line

Can you be a bit more specific about which file you're referring to? Or you can also make a comment directly on the line in the PR :)

File: earthquakes/python/publisher.py [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
 
 import requests
 from pyensign.events import Event
 from pyensign.ensign import Ensign
 
-# TODO Python>3.10 needs to ignore DeprecationWarning: There is no current event loop
-warnings.filterwarnings("ignore")
-
 # TODO: replace with YOU - your email and app details :)
 ME = "(https://rotational.io/data-playground/us-geological, earthquakes@rotational.io)"
```
Comment (Aatmaj1) [inline earthquakes/python/publisher.py:11]: Should we add comments why this is being done ? why your email and app details are provided here ?

Comment (pdeziel) [reply earthquakes/python/publisher.py:11]: Sure, I've added a comment.

**Review by Aatmaj1 — APPROVED** [review]

**Pull request merged**